web/src/app/components/social-cards/social-cards.component.html

```diff
@@ -0,0 +1,93 @@
+<mat-card class="outer-container" fxLayout="row" fxLayoutGap="20px">
+  <mat-card fxFlex="25">
+    <mat-card-content *ngIf="userSocial.github != undefined else github">
+      <mat-icon svgIcon="github" class="github"></mat-icon>
+      <div fxLayout class="stats-wrapper">
+        <div fxLayout="column" fxFlex="33">
+          <span class="stats-display">{{userSocial.github.followers}}</span>
+          <span class="stats-label">Followers</span>
+        </div>
+        <div fxLayout="column" fxFlex="33">
+          <span class="stats-display">{{userSocial.github.following}}</span>
+          <span class="stats-label">Following</span>
+        </div>
+      </div>
+    </mat-card-content>
+    <ng-template #github>
```

Comment:
I think it would be simpler if on line 3 we used `*ngIf="userSocial.github"` and then here use the opposite `*ngIf="!userSocial.github"`

Reply:
Okay. Have changed it.